Git-commit: 45cd7db0d17cb91feff1ce2f296266354289b713
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 19 Jan 2021 08:57:21 +0000
Subject: [PATCH] configs: Enable BCM2835 thermal driver in kernel8
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

The arm64 version of bcm2711_defconfig is intended for Pi 4, but the
Raspberry Pi OS kernel8.img built from it is also used for Pi 3. It is
therefore necessary to include some legacy drivers such as the BCM2835
thermal driver, otherwise there is no support for reading the CPU
temperature on a Pi 3 in 64-bit mode.

See: https://github.com/raspberrypi/linux/issues/4077

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/configs/bcm2711_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 325ff25a243d..eeb52d96e26b 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -724,6 +724,7 @@ CONFIG_SENSORS_INA2XX=m
 CONFIG_SENSORS_TMP102=m
 CONFIG_THERMAL=y
 CONFIG_BCM2711_THERMAL=y
+CONFIG_BCM2835_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_GPIO_WATCHDOG=m
 CONFIG_BCM2835_WDT=y
-- 
2.31.1

